src/gateway/get-started/get-kong.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Get Kong
+content-type: tutorial
+---
+
+[{{site.base_gateway}}](/gateway/latest/) is a lightweight, fast, and flexible cloud-native API gateway. 
+{{site.base_gateway}} sits in front of your service applications, dynamically controlling, analyzing, and 
+routing requests and responses. {{site.base_gateway}} implements your API traffic policies 
+by using a flexible, low-code, plug-in based approach. 
+
+This tutorial will help you get started with {{site.base_gateway}} by setting up a local installation
+and walking through some common API management tasks. 
+
+This page will walk you through running {{site.base_gateway}} and verifying it with the
+[Admin API](/gateway/latest/admin-api). Once complete, the following tasks 
+can be performed to complete the tutorial:
+
+* [Understanding and configuring Services and Routes](/gateway/latest/get-started/configure-services-and-routes)
+* [Configuring Rate Limiting to protect upstream Services](/gateway/latest/get-started/configure-ratelimiting)
+* [Increase system performance with Proxy Caching](/gateway/latest/get-started/proxy-caching)
+* [Load Balancing for horizontal Service scaling](/gateway/latest/get-started/load-balancing)
+* [Protecting Services with Key Authentication](/gateway/latest/get-started/protect-services)
+* [Use Workspaces and Teams to further protect {{site.base_gateway}}](/gateway/latest/get-started/manage-teams)
```

Comment:
```suggestion
* [Understanding and configuring Services and Routes](/gateway/latest/get-started/configure-services-and-routes)
* [Configuring Rate Limiting to protect upstream Services](/gateway/latest/get-started/configure-ratelimiting)
* [Increase system performance with Proxy Caching](/gateway/latest/get-started/proxy-caching)
* [Load Balancing for horizontal Service scaling](/gateway/latest/get-started/load-balancing)
* [Protecting Services with Key Authentication](/gateway/latest/get-started/protect-services)
* [Use Workspaces and Teams to organize {{site.base_gateway}}](/gateway/latest/get-started/manage-teams) at scale.
```

Reply:
I took this from our marketing site, I felt it was a bit stronger than 'further protect'.